auditwheel/lddtree.py

```diff
@@ -195,11 +198,31 @@ def load_ld_paths(root: str = '/', prefix: str = '') -> Dict[str, List[str]]:
             # on a per-ELF basis so it can get turned into the right thing.
             ldpaths['env'] = parse_ld_paths(env_ldpath, path='')
 
-    # Load up /etc/ld.so.conf.
-    ldpaths['conf'] = parse_ld_so_conf(root + prefix + '/etc/ld.so.conf',
-                                       root=root)
-    # the trusted directories are not necessarily in ld.so.conf
-    ldpaths['conf'].extend(['/lib', '/lib64/', '/usr/lib', '/usr/lib64'])
+    libc = get_libc()
+    if libc == Libc.MUSL:
+        # from https://git.musl-libc.org/cgit/musl/tree/ldso
+        # /dynlink.c?id=3f701faace7addc75d16dea8a6cd769fa5b3f260#n1063
+        ld_musl = list(Path(root + prefix + '/etc').glob("ld-musl-*.path"))
+        assert len(ld_musl) <= 1
+        if len(ld_musl) == 0:
+            ldpaths['conf'] = [
+                root + '/lib',
+                root + '/usr/local/lib',
+                root + '/usr/lib'
+            ]
+        else:
+            ldpaths['conf'] = []
+            for ldpath in ld_musl[0].read_text().split(':'):
+                ldpath_stripped = ldpath.strip()
+                if ldpath_stripped == "":
+                    continue
+                ldpaths['conf'].append(root + ldpath_stripped)
+    else:
+        # Load up /etc/ld.so.conf.
+        ldpaths['conf'] = parse_ld_so_conf(root + prefix + '/etc/ld.so.conf',
```

Comment:
```suggestion
        ldpaths['conf'] = parse_ld_so_conf(str(root_prefix / 'etc/ld.so.conf'),
```

Reply:
I'd rather not touch the glibc code path for now. We can get this cleaned up later.